apps/dashboard/src/components/workflow-editor/steps/digest/digest-control-values.tsx

```diff
@@ -0,0 +1,42 @@
+import { UiSchemaGroupEnum } from '@novu/shared';
+
+import { getComponentByType } from '@/components/workflow-editor/steps/component-utils';
+import { useStep } from '@/components/workflow-editor/steps/step-provider';
+import { Separator } from '@/components/primitives/separator';
+import { SidebarContent } from '@/components/side-navigation/sidebar';
+
+export const DigestControlValues = () => {
+  const { step } = useStep();
+  const { uiSchema } = step?.controls ?? {};
+
+  if (!uiSchema || uiSchema?.group !== UiSchemaGroupEnum.DIGEST) {
+    return null;
+  }
+
+  const { ['amount']: amount, ['digestKey']: digestKey, ['unit']: unit, ['cron']: cron } = uiSchema.properties ?? {};
+
+  return (
+    <div className="flex flex-col">
+      {digestKey && (
+        <>
+          <SidebarContent size="lg">
+            {getComponentByType({
+              component: digestKey.component,
+            })}
+          </SidebarContent>
+          <Separator />
```

Comment:
Ditto about layout JSX leaking into the digest-control-values component.

Reply:
Answered in the previous comment.